taxi/models.py

```diff
@@ -0,0 +1,28 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+    country = models.CharField(max_length=63)
```

Comment:
Increase `max_length` to 255 characters, 63 is a low limitation and unnecessary here.